### ISSUE TITLE
Title of the elemental area in the CMS?

### ISSUE THREAD

**Issue by brasileric** [dnadesign/silverstripe-elemental#256]
Hi,
I have multiple elemental areas on one page, but the title doesn't appear in the CMS. How can I set the title of each area?

My code:
````    private static $has_one = array(
        'ContentArea' => ElementalArea::class,
        'PanelArea' => ElementalArea::class
    );

    private static $owns = array(
        'ContentArea',
        'PanelArea'
    );
````
But in the CMS just the 2 areas without title above:

![image](https://user-images.githubusercontent.com/6453793/42640076-c3fe384e-85f1-11e8-8b85-71f76da77f86.png)

**Comment by NightJar** [comment]
The `ElementalArea` CMS field is added via the `ElementalAreasExtension` (or it's subclass `ElemetnalPageExtension`). This is via a factory `ElementalArea::getField` which defaults to to a title of `''` by default. This factory has a `setTitle` function, but accessing it may seem be overly complicated... however it returns a normal `GridField` so `setTitle()` is all that's needed.

tl;dr - from `getCMSFields` or `updateCMSFields` via an extension:
`$form->fieldByName($relationshipID)->setTitle($relationshipName);`

In either case this is intentional so I'll close this as not a bug. But I hope this guides you to an answer :)
